core/src/test/java/org/springframework/security/core/userdetails/PasswordEncodedUser.java

```diff
@@ -25,12 +25,18 @@
 public class PasswordEncodedUser {
 	private static final UserDetails USER = withUsername("user").password("password").roles("USER").build();
 
+        private static final UserDetails USER_NOT_LOWERCASE = withUsername("User").password("password").roles("USER").build();
```

Comment:
Sorry. I think you misunderstood what I was asking. Rather than creating a new public user in here. Just create one in the test itself.

Reply:
This will be removed, but in general please pay attention to whites pace. This is incorrectly indented with spaces rather than tabs which means the checkstyle tests will fail. Please use tabs.

Reply:
Ok, Thanks.